src/components/SidebarTabs/SharingSidebarTab.vue

```diff
@@ -232,6 +249,7 @@ export default {
 		return {
 			isLoading: false,
 			appConfig: loadState(appName, 'appConfig'),
+			qrDialog: false,
```

Comment:
```suggestion
			qrDialogText: null,
```

Mixing boolean and string here is not really obvious I think, so instead make it string by default with `null` to explicitly mark *no value*.